src/NHapi.Base/SupportClass.cs

```diff
@@ -1,3268 +1,3486 @@
+            }
+            else
+            {
+                return string.Empty;
+            }
+        }
+
+        /*******************************/
+
+        /// <summary>
+        /// This class is created to save the information of each attributes in the SaxAttributesSupport.
+        /// </summary>
+        public class Att_Instance
+        {
+            public string att_URI;
+            public string att_localName;
+            public string att_fullName;
+            public string att_type;
+            public string att_value;
+
+            /// <summary>
+            /// This is the constructor of the Att_Instance.
+            /// </summary>
+            /// <param name="Uri">The namespace URI of the attribute.</param>
+            /// <param name="Lname">The local name of the attribute.</param>
+            /// <param name="Qname">The long(Qualify) name of attribute.</param>
+            /// <param name="Type">The type of the attribute.</param>
+            /// <param name="Value">The value of the attribute.</param>
+            public Att_Instance(string Uri, string Lname, string Qname, string Type, string Value)
+            {
+                att_URI = Uri;
+                att_localName = Lname;
+                att_fullName = Qname;
+                att_type = Type;
+                att_value = Value;
+            }
+        }
+    }
+
+    /*******************************/
+
+    /// <summary>
+    /// This exception is thrown by the XmlSaxDocumentManager in the SetProperty and SetFeature
+    /// methods if a property or method couldn't be found.
+    /// </summary>
+    public class ManagerNotRecognizedException : Exception
+    {
+        /// <summary>
+        /// Creates a new ManagerNotRecognizedException with the message specified.
+        /// </summary>
+        /// <param name="Message">Error message of the exception.</param>
+        public ManagerNotRecognizedException(string Message)
+            : base(Message)
+        {
+        }
+    }
+
+    /*******************************/
+
+    /// <summary>
+    /// This exception is thrown by the XmlSaxDocumentManager in the SetProperty and SetFeature methods
+    /// if a property or method couldn't be supported.
+    /// </summary>
+    public class ManagerNotSupportedException : Exception
+    {
+        /// <summary>
+        /// Creates a new ManagerNotSupportedException with the message specified.
+        /// </summary>
+        /// <param name="Message">Error message of the exception.</param>
+        public ManagerNotSupportedException(string Message)
+            : base(Message)
+        {
+        }
+    }
+
+    /*******************************/
+
+    /// <summary>
+    /// This class provides the base implementation for the management of XML documents parsing.
+    /// </summary>
+    public class XmlSaxDefaultHandler : IXmlSaxContentHandler, IXmlSaxErrorHandler, IXmlSaxEntityResolver

```

Comment:
this doesnt even have an implamentation as such? 😫

Reply:
will log a separate issue to refactor this `SupportClass`